WordPress/Docs/PHP/WordPress.WP.AlternativeFunctions.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="WordPress Alternative Functions"
+    >
+    <standard>
+    <![CDATA[
+    Use WordPress functions instead of native PHP functions to maintain compatibility and benefit from WordPress's additional security and performance improvements.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Using wp_safe_redirect().">
+        <![CDATA[
+        wp_safe_redirect( $url );
+        ]]>
+        </code>
+        <code title="Invalid: Using PHP's header() function for redirection.">
+        <![CDATA[
+        header( "Location: $url" );
```

Comment:
Please use `<em>` tags to highlight the "good" and the "bad" bits in the code examples. More information in the description of #1722.